File: samples_generated_code/samples_2/phi4/baseline_code_c/code_row_60.c
#include <stdio.h>

int get_value_at_index() {
    int id_sequence[] = {1, 2, 3, 4, 5}; // Example array, modify as needed
    int size = sizeof(id_sequence) / sizeof(id_sequence[0]);
    
    int index;
    printf("Enter an index value: ");
    scanf("%d", &index);
    
    if (index >= 0 && index < size) {
        return id_sequence[index];
    } else {
        printf("Index out of bounds\n");
        return -1; // Indicate error
    }
}

int main() {
    int result = get_value_at_index();
    if (result != -1) {
        printf("Value at the given index: %d\n", result);
    }
    return 0;
}

File: samples_generated_code/samples_2/phi4/baseline_code_py/code_row_60.py
def get_value_at_index():
    import sys

    id_sequence = [10, 20, 30, 40, 50]

    try:
        index_input = int(input("Enter an index value: "))
        if index_input < 0 or index_input >= len(id_sequence):
            print(f"Index out of range. Please enter a value between 0 and {len(id_sequence) - 1}.")
            return None
        return id_sequence[index_input]
    except ValueError:
        print("Invalid input. Please enter an integer.")
        sys.exit(1)

# Example usage:
if __name__ == "__main__":
    result = get_value_at_index()
    if result is not None:
        print(f"Value at the entered index: {result}")